xbmc/cores/VideoPlayer/VideoRenderers/windows/RendererDXVA.cpp

```diff
@@ -295,6 +300,16 @@ CRenderBuffer* CRendererDXVA::CreateBuffer()
   return new CRenderBufferImpl(m_format, m_sourceWidth, m_sourceHeight);
 }
 
+std::string CRendererDXVA::GetRenderMethodDebugInfo() const
+{
+  if (m_processor && DX::Windowing()->SupportsVideoSuperResolution())
+  {
+    return StringUtils::Format("Video Super Resolution: {}",
+                               m_processor->IsVideoSuperResolutionEnabled() ? "enabled" : "OFF");
```

Comment:
Sorry, now thinking that as final decision on Super Resolution enabled or not is in Nvidia driver and driver settings (Kodi only request enable) and is not possible confirm if is enabled or not (no API available yet) is better change this to:

`requested enable`

to avoid users confusion

Reply:
that's fine, I didn't use ON for that reason. Just "requested" is enough I think.